CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+This document is a work in progress.
+
+*Things to cover: pipenv, Cloudformation, Sceptre, SAM, Github workflows, git hooks*
+
+### Monitoring
+
+Use a combination of Workflows and Glue Studio, both within the [Glue console](https://console.aws.amazon.com/glue/), for graphical monitoring. Workflows gives a good view of what's happening within a workflow, but for a better overview of all the job runs, Glue Studio is recommended. Either option will give you a link to the Cloudwatch logs for each job.
```

Comment:
A fourth monitoring option will be the Cloudwatch metrics, but I'm still working on those.